opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -527,6 +527,17 @@ void VFPProdTable::convertALQToSI(const ALQ_TYPE& type,
 
 
 
+const MessageContainer& VFPProdTable::getMessageContainer() const
+{
+    return m_messages;
+}
+
+
+
+MessageContainer& VFPProdTable::getMessageContainer()
```

Comment:
Is this mutable reference really needed - if not I would prefer if it were removed.